### PR TITLE
Add reminder service

### PR DESCRIPTION
**Pull request by tjerman**
Interested app should register this plugin to handle
reminder pooling.

### CODE REVIEW

File: src/components/Toaster/index.vue [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+    evtSink () {},
+  },
+}
+</script>
+
+<style scoped lang="scss">
+// @note Couldn't get this bootstrap prop to work, so here it is for now.
+.border-transparent {
+  border-color: transparent!important;
```
Comment (zmija) [inline src/components/Toaster/index.vue]: what if you use ```border-0```class instead?

File: src/components/Toaster/index.vue [hunk 2]
```diff
@@ -73,3 +74,10 @@ export default {
   border-color: transparent!important;
 }
 </style>
+
+<style lang="scss">
+// BS only provides overflow-[auto, hidden]
+.toast.overflow-unset {
```
Comment (zmija) [inline src/components/Toaster/index.vue]: put it inside scoped scss and use ```/deep/```

Comment (tjerman) [reply src/components/Toaster/index.vue]: That was my initial plan but there is a slight problem. Bootstrap doesn't show it's toasts in this component, but rather portals them into its own toaster component that is located somewhere else.

( with this: https://www.npmjs.com/package/portal-vue )

I'll take another look (later) if I can figure out something scoped.

Comment (zmija) [reply src/components/Toaster/index.vue]: if you'll remove ```.border-transparent``` class from scoped scss (and I'm pretty sure it's necessary as you can use ```.border-0```) then it's ok to leave this one as it is.

File: src/components/Toaster/index.vue (hunk 1 above)
Comment (tjerman) [reply src/components/Toaster/index.vue]: Fair point for next time; this one is not used here anymore

**Pull request merged**